openslides_backend/action/actions/user/save_saml_account.py

```diff
@@ -465,10 +459,26 @@ def get_field_data(
                             )
                         )
                     elif saml_meeting_user_field == "vote_weight":
-                        # Result is string and has 6 digits.
-                        if isinstance(value, str):
-                            value = float(value)
-                        result = f"{value:f}"
+                        # Result must be string and have 6 digits after dot.
+                        if isinstance(value, int):
+                            value = f"{value:f}"
+                        elif isinstance(value, str) and re.compile(
+                            r"^[1-9]\d*\.?\d{0,6}$"
+                        ).match(value):
```

Comment:
What is the rationale behind throwing numbers with 7 or more digits after the point away?

Reply:
As I was told by @rrenkert that we only support numbers with up to 6 digits precision. Everything else must be handled as an error by the user/idp. This is also ensured by the field validation checks where exactly 6 digits after the dot are the accepted.

Reply:
Okay